### PR TITLE
Friends specs

### PR DESCRIPTION
**Pull request by flaming-**

### CODE REVIEW

**Comment by jimrollenhagen** [comment]
@flaming- as a note, you can update a PR with either 1) a new commit on the branch, or 2) amending your commit and force pushing. No need to close a PR and open a new one. In fact, I'd prefer you didn't as we lose the discussion.

Reviewers: see also discussion on #109.

File: doc/specs/friends.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+* User X's saved notes on Y are updated to reflect what User X entered in the Notes box
+
+**Basic Flow:**
+
+1. User edits the "Notes" text box pertaining to user Y
+1. User clicks the "Update" button next to this text box
+
+**Applicable business rules:**
+
+* This action does not notify User Y in any way
```
Comment (jimrollenhagen) [inline doc/specs/friends.md:114]: Needs moar data model.

Comment (ghost) [inline doc/specs/friends.md:114]: No more data model in specs.

**Comment by ghost** [comment]
@flaming- You need to be more explicit with who has and needs what permissions. Reference some of the existing specs. 

+1 besides that.

**Comment by ghost** [comment]
@flaming- Can this be merged?

**Comment by flaming-** [comment]
@ajax-wcd I will add an explicit use case for admins disabling friend system permissions, this should address the "who has and needs what permissions" concern I believe, then it can be merged. (I cannot get on this earlier than tonight.)

**Comment by ghost** [comment]
@flaming- Great, @ me on here so I don't miss it

**Comment by flaming-** [comment]
@ajax-wcd actually, I think the specifics of permissions modification belong in the permissions spec. I have added an explicit mention of a permissions flag (I believe there is only one relevant permission here). Is this what you meant by being "more explicit with who has and needs what permissions" or did you have something else in mind. If this looks good to you then it is ready to be merged.

Also, in the future, if we have to make edits to pull request branches like this, would you prefer we just leave it as two commits, or rebase them and --force push to make it appear as one commit on the branch? I seem to remember you being somewhat particular about rebasing.

**Comment by ghost** [comment]
@flaming- Take a look at how permissions are done in other specs to see what I mean. Announcements spec for example.

It doesn't matter how you update your branch.

**Comment by ghost** [comment]
@flaming- What's going on with this?

**Comment by ghost** [comment]
I'm taking this over.

**Comment by ghost** [comment]
@ajax-wcd Does this need more information regarding permissions or is this ready for merge? (I'm going to remove WCD specific stuff)

**Pull request closed without merging**